### ISSUE TITLE
DatePicker A11y: All content must reside within a WAI-ARIA landmark or labelled region role

### ISSUE THREAD

**Issue by rstrangfeld** [IBM/carbon-components-react#2125]
When using the DatePicker, DAP reports the violation 
"All content must reside within a WAI-ARIA landmark or labelled region role." It associates this violation with the first input element within the calendar, the numInput for the current year.

Looking in the DOM in browser tools, the application code puts a DatePicker within normal content inside a div having role="region", but the elements that make up the calendar appear within a div having class "flatpickr-calendar" immediately below the body, with no labelled role. 

Browser: Chrome 73.0.3683.86.
Carbon versions: carbon-components-react 6.71.0 and carbon-components 9.62.3
Offering/product: IBM MDM Express.

**Comment by rstrangfeld** [comment]
It looks like stale bot is threatening to close this issue because of inactivity. We're still interested in a fix. What would need need to happen for that to come about?
Thanks!

**Comment by dakahn** [comment]
Please create a reduced test case in CodeSandbox
https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox
